src/env.h

```diff
@@ -322,7 +322,7 @@ constexpr size_t kFsStatsBufferLength =
   V(promise_string, "promise")                                                 \
   V(pubkey_string, "pubkey")                                                   \
   V(query_string, "query")                                                     \
-  V(quic_alpn_string, "h3-22")                                                 \
+  V(quic_alpn_string, "h3-24")                                                 \
```

Comment:
While working on #200 I ran into 

```
=== release test-quic-binding ===                   
Path: parallel/test-quic-binding

+ 'ff000018'
- 'ff000017'
```

Is that something that needs to be updated in this PR too?

Reply:
Yes. Will do.